ocs_ci/ocs/fiojob.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf8 -*-
+
+"""
+This module contains functions which implements functionality necessary to run
+general `fio`_ workloads as `k8s Jobs`_ in OCP/OCS cluster via workload
+fixtures (see :py:mod:`ocs_ci.utility.workloadfixture`).
+
+.. moduleauthor:: Martin Bukatovič
+
+.. _`fio`: https://fio.readthedocs.io/en/latest/fio_doc.html
+.. _`k8s Jobs`: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/
+"""
+
+
+import logging
+import time
+import yaml
+import yaml.parser
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_storageutilization_size(target_percentage, ceph_pool_name):
+    """
+    For the purpose of the workload storage utilization fixtures, get expected
+    pvc_size based on STORED and MAX AVAIL values (as reported by `ceph df`)
+    for given ceph pool and target utilization percentage.
+
+    This is only approximate, and it won't work eg. if each pool has different
+    configuration of replication.
+
+    Args:
+        target_percentage (float): target total utilization, eg. 0.5 for 50%
+        ceph_pool_name (str): name of ceph pool where you want to write data
+
+    Returns:
+        int: pvc_size for storage utilization job (in GiB, rounded)
```

Comment:
One empty line after last section

Reply:
Why?

Reply:
Cause it's good practice followed also in sphinx docstring examples:  https://www.sphinx-doc.org/en/1.5/ext/example_google.html

After each section like Args Returns should be one empty line.

Reply:
BTW we are pointing in our docs as well to follow this example:
https://ocs-ci.readthedocs.io/en/latest/docs/coding_guidelines.html

> Each Python Class is well documented for inheritance, methods and attributes. Add sufficient comments for code block that is easy to understand for others to review. Use Google Style of Python Docstrings

pointing to https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html where you can see is the empty line followed after each sections.

Reply:
I had hoped you will be able to provide to a particular statement about this newline, maybe even with some reasoning. It's easy to miss in the canonical example you linked.

Reply:
Based on your feedback, I almost expected that there is some implementation detail I'm missing ...

Reply:
I remember I got the same review about missing empty line comment a lot of years back in RHV from someone and IIRC he pointed that time that if the empty line is missing the sphinx had some issue with rendering the content in doc we generated. (We used different tool to render the doc that time.)

Since that, the issue maybe got fixed and sphinx is maybe ignoring this and have no issue at all with that but I am trying to follow the example in official doc where they use this space as well.